src/components/Categories.js

```diff
@@ -1,8 +1,17 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
 function Categories() {
+  const state = useSelector((state) => state.categoriesReducer);
+  const dispatch = useDispatch();
   return (
-    <button type="button">Check status</button>
+    <div>
+      <h2>{state}</h2>
+      {' '}
+
+      <button type="button" onClick={() => dispatch(checkStatus())}>Check status</button>
```

Comment:
- [ ] Based on the requirements of this project, this button should use a function `checkCategoriesStatus`(named as you wish) made in this component. The functions should dispatch the `checkStatus` action creator. Let's kindly implement this function inside this component.